invenio_cli/helpers/cli_config.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019-2020 CERN.
+# Copyright (C) 2019-2020 Northwestern University.
+#
+# Invenio-Cli is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio-cli configuration file."""
+
+import os
+from configparser import ConfigParser
+from pathlib import Path
+
+from .filesystem import get_created_files
+
+
+class CLIConfig(object):
+    """Invenio-cli configuration."""
+
+    CONFIG_FILENAME = '.invenio'
+    CLI_SECTION = 'cli'
+    COOKIECUTTER_SECTION = 'cookiecutter'
+    FILES_SECTION = 'files'
+
+    def __init__(self, fullpath=CONFIG_FILENAME, verbose=False):
+        """Constructor.
+
+        :param flavour: Flavour name.
+        """
+        self.config = ConfigParser()
+        self.fullpath = fullpath
+        self.config.read(fullpath)
+
+    def get_project_dir(self):
+        """Returns path to project directory."""
+        return Path(self.config[CLIConfig.CLI_SECTION]['project_dir'])
+
+    def get_instance_path(self):
+        """Returns path to application instance directory."""
+        return Path(self.config[CLIConfig.CLI_SECTION]['instance_path'])
+
+    def update_instance_path(self, new_instance_path):
+        """Updates path to application instance directory."""
+        self.config[CLIConfig.CLI_SECTION]['instance_path'] = \
+            str(new_instance_path)
+
+        with open(self.fullpath, 'w') as configfile:
+            self.config.write(configfile)
+
+    @classmethod
+    def write(cls, project_dir, flavour, replay):
+        """Write invenio-cli config file.
+
+        :param project_dir: Folder to write the config file into
+        :param flavour: 'RDM' or 'ILS'
+        :param replay: dict of cookiecutter replay
+        :return: full path to config file
+        """
+        config_parser = ConfigParser()
+
+        # Internal to Invenio-cli section
+        config_parser[cls.CLI_SECTION] = {}
```

Comment:
@fenekku I think here you miss the `project_shortname` information :) I tried to run the cli locally but when it finishes the build and I am trying to start the server it requires to read the `project_shortname` from `cli` section in `.invenio.cfg` but it's not there. Do I miss something? Could you try it too and see if ti happens for you too?

Reply:
Thanks for noticing this. Yes I explicitly removed it, because I will be using the value in the "cookiecutter" section for that instead (rather than saving it twice in the file). I will add it back in this PR and then remove it, once I get to the running part.